### DISCUSSION TITLE
Megane II ph1 card programming

### DISCUSSION THREAD

**Discussion by Hase0** [cedricp/ddt4all#1216] [Q&A]
I tried to add new card but I'm stuck at this problem. How can I resolve this? 
![IMG_20240826_070623](https://github.com/user-attachments/assets/e5ba2623-362a-4b59-8fc7-f784928d83c3)

**Comment by Asphiii** [comment]
@Hase0 do you have fix? I have the very same issue in my car

**Comment by martinkoenig** [comment]
Same for me
